### PR TITLE
fix(web): redundant codes and type assertion

### PR DESCRIPTION
**Pull request by caichi-t**
## What I've done
- Fixed the sort operator initialization issue
- Fixed the redundant codes
- Avoided using assertions as much as possible and added processes of type judgments.
## Memo
In some cases, it may be better to review the original type definitions and reduce the processes of type judgments.

### CODE REVIEW

File: web/src/components/molecules/Content/Table/index.tsx [hunk 1]
```diff
@@ -258,57 +226,20 @@ const ContentTable: React.FC<Props> = ({
...
-  );
+    ];
+  }, [t, currentView.sort, selectedItem?.id, onItemSelect]);
 
   const tableColumns = useMemo(() => {
-    const items = [...actionsColumns];
-    if (contentColumns) items.push(...contentColumns);
-    return items;
-  }, [actionsColumns, contentColumns]);
+    return contentTableColumns ? [...actionsColumns, ...contentTableColumns] : actionsColumns;
```
Comment (nourbalaha) [inline web/src/components/molecules/Content/Table/index.tsx:241]:
```suggestion
    return contentTableColumns ? [...actionsColumns, ...contentTableColumns] : [...actionsColumns];
```

File: web/src/components/molecules/Content/Table/index.tsx [hunk 2]
```diff
@@ -362,37 +293,51 @@ const ContentTable: React.FC<Props> = ({
...
-          if (c.key === fieldId.id) {
-            column = c;
-            break;
+        const column = (() => {
+          for (const c of columns) {
+            if (c.key === fieldId.id) {
+              return c;
+            }
+          }
+        })();
```
Comment (nourbalaha) [inline web/src/components/molecules/Content/Table/index.tsx:314]:
```suggestion
        const column = columns.find(c => c.key === fieldId.id);
```

File: web/src/components/molecules/Content/Table/index.tsx [hunk 3]
```diff
@@ -694,33 +646,38 @@ const ContentTable: React.FC<Props> = ({
...
+            if (
+              col.key === "EDIT_ICON" ||
+              col.key === "commentsCount" ||
+              hiddenCols.includes(col.key)
+            ) {
+              return false;
+            } else {
+              return true;
+            }
+          }
```
Comment (nourbalaha) [inline web/src/components/molecules/Content/Table/index.tsx:659]:
```suggestion
          return typeof col.key === "string" &&
              col.key !== "EDIT_ICON" &&
              col.key !== "commentsCount" &&
              !hiddenCols.includes(col.key)
```

**Review by nourbalaha — CHANGES_REQUESTED** [review]

File: web/src/components/molecules/Content/Table/index.tsx (hunk 2 above)
Comment (caichi-t) [reply web/src/components/molecules/Content/Table/index.tsx:314]: Thank you! I wrote the weird code. lol

**Review by nourbalaha — APPROVED** [review]

**Pull request merged**